pkgs/applications/editors/edit/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, unzip, pkgconfig, ncurses, x11, cwebbin }:
```

Comment:
So should I change it to `xlibsWrapper` or is the current (libx11 + libxft) OK? it seems to be passing in hydra